test.js

```diff
@@ -447,3 +451,50 @@ test('alias - require previousId and userId', t => {
     })
   })
 })
+
+test('e2e', async t => {
```

Comment:
I still think this should only run in certain conditions.

Maybe do something like:

```js
const { RUN_E2E_TESTS } = process.env
if (RUN_E2E_TESTS) {
  test('e2e', async t => { ... })
}
```

And in `circle.yml`, add `RUN_E2E_TESTS=true` in the "environment" config.

Reply:
To avoid nesting, you can do this:

```js
async function e2etest (t) {
  const id = uid(16)
  // ...
}

const { RUN_E2E_TESTS } = process.env
if (RUN_E2E_TESTS) {
  test('e2e', e2etest)
}

Reply:
Yep, didn't get around to that yet. Will update on the original comment when I do.

Reply:
Nesting here is totally fine, looks more natural.

Reply:
@f2prateek could you also modify test title to be more descriptive?

Reply:
Sure, do you have any other suggestions?

Reply:
`send event to runscope`

Reply:
I updated the description and added some comments, let me know if that looks ok.

Reply:
done